docs/layouts/partials/icon.html

```diff
@@ -1,2 +1,2 @@
 <span class="icon material-symbols-{{ default "outlined" .Style }} {{ .Class }}"
-      aria-label="{{ .Label }}" aria-hidden="{{ default "false" .Hidden }}">{{ .Icon }}</span>
+      aria-label="{{ .Label }}" aria-hidden="{{ default "true" .Hidden }}">{{ .Icon }}</span>
```

Comment:
Would having it always be hidden with some `.sr-only` text instead of a label make more sense?
```suggestion
      aria-hidden="true">{{ .Icon }}</span>
{{ with .Label }}
    <span class="sr-only">{{ . }}</span>
{{ end }}
```
This suggestion also results in it being completely hidden by default unless a label is specified.